master/usage/removing-nodes.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Removing Nodes
+---
+
+With each Calico node there is a Node entry in the datastore and Workload
+Endpoints are associated with that node.  As nodes are replaced (with
+differently named nodes) or decomissioned, the datastore will accumulate
```

Comment:
True.  I was talking to someone that had to do name changes due to internal DNS changes and so that is kind of what I was thinking with that statement.